lib/ethereum/contract.rb

```diff
@@ -185,11 +193,23 @@ def call(fun, *args)
       end
     end
 
+    def encode_tx(tx_args, fun, *args)
```

Comment:
I am not sure what encode_tx suppose to do. If it only encodes transaction than we should not send it? Right?